workspace/dao.py

```diff
@@ -0,0 +1,73 @@
+from db_classes import Workspace
+
+def create(name, owner):
+    """
+    Create a new empty Workspace
+    """
+
+    workspace = Workspace(name = name, users = [owner], task_types = [], tasks = [])
```

Comment:
I wonder if we should add the default task type here or if we should just always assume it is globally available (I think the former might make things simpler long-term though)